app/assets/javascript/components/editor/editor.js

```diff
@@ -127,13 +128,46 @@ const EditorController = class extends Controller {
     });
   }
 
-  static contentWrapper = (content) => {
-    const strippedContent = content.replace('<editor-content>', '').replace('</editor-content>', '');
-    return `<editor-content>${strippedContent}</editor-content>`;
-  };
+  wordCount(content) {
```

Comment:
gonna tidy all this up in terms of code style. maybe make the word count stuff a separate script with tests. i dont feel it should be encapsuated in an editor component quite like this and needs decoupling from it